whylog_vim/exceptions.py

```diff
@@ -0,0 +1,12 @@
+class WhylogVIMExceptions():
+    pass
+
+
+class UnknownAction(WhylogVIMExceptions):
+
+    def __init__(self, move_type, state):
+        self.move_type = move_type
+        self.state = state
+
+    def __repr__(self):
+        'Unknown action for %s in states %s' % (self.move_type, self.state)
```

Comment:
Please add a name before first argument occurrence
Unknown action for %s -> Unknown action for argument_name %s